templates/test/ci/prow-machine-pool-ci-version/patches/machine-pool-ci-version.yaml

```diff
@@ -47,16 +46,23 @@ spec:
               DEBIAN_FRONTEND=noninteractive apt-get install -y $$CI_PACKAGE=$$PACKAGE_VERSION
             done
           else
-              CI_URL="https://storage.googleapis.com/k8s-release-dev/ci/$${CI_VERSION}/bin/linux/amd64"
+            CI_URL="https://storage.googleapis.com/k8s-release-dev/ci/$${CI_VERSION}/bin/linux/amd64"
             fi
             for CI_PACKAGE in "$${PACKAGES_TO_TEST[@]}"; do
               echo "* downloading binary: $$CI_URL/$$CI_PACKAGE"
               wget "$$CI_URL/$$CI_PACKAGE" -nv -O "$$CI_DIR/$$CI_PACKAGE"
               chmod +x "$$CI_DIR/$$CI_PACKAGE"
               mv "$$CI_DIR/$$CI_PACKAGE" "/usr/bin/$$CI_PACKAGE"
             done
-            systemctl restart kubelet
+            for CI_CONTAINER in "$${CONTAINERS_TO_TEST[@]}"; do
+              echo "* downloading package: $$CI_URL/$$CI_CONTAINER.$$CONTAINER_EXT"
+              wget "$$CI_URL/$$CI_CONTAINER.$$CONTAINER_EXT" -nv -O "$$CI_DIR/$$CI_CONTAINER.$$CONTAINER_EXT"
+              $${SUDO} ctr -n k8s.io images import "$$CI_DIR/$$CI_CONTAINER.$$CONTAINER_EXT" || echo "* ignoring expected 'ctr images import' result"
+              $${SUDO} ctr -n k8s.io images tag registry.k8s.io/$$CI_CONTAINER-amd64:"$${CI_VERSION//+/_}" registry.k8s.io/$$CI_CONTAINER:"$${CI_VERSION//+/_}"
```

Comment:
@sbueringer makes a good point in https://kubernetes.slack.com/archives/CEX9HENG7/p1655744028320889?thread_ts=1655475223.796599&cid=CEX9HENG7 that this likely breaks for k8s < 1.25

Reply:
@CecileRobertMichon do you mean >= 1.25? (we've already verified that it works fine for 1.24 and below)

Reply:
Oh I see what you mean, we're saying that these changes are only valid now and into the future, but not if we use against commit checkouts from the past.

Reply:
Pushed up a follow-up change that (I think) is a little cleaner to maintain, and tested out as working in both of the following scenarios:

```
$ KUBERNETES_VERSION="latest" CONTROL_PLANE_MACHINE_COUNT="3" ./scripts/ci-entrypoint.sh
```

```
CONTROL_PLANE_MACHINE_COUNT="3" CI_VERSION="v1.24.0-alpha.4.102+99e36a93b21702" ./scripts/ci-entrypoint.sh
```

i.e., the new and old ways of doing things, respectively